packages/kotti-ui/source/kotti-field/mixins.scss

```diff
@@ -438,7 +438,7 @@
 			/* stylelint-disable */
 			font-size: map.get($fontSizes, $type);
 
-			.kt-field__input-container {
```

Comment:
This seems pretty random

Reply:
it's not, all is related to when you have two `.kt-fields_input-container` that can be influened by the same specifity 

happens when you use the datetime field and datetime range inside popper because the parent becomes the field itself and affects the `input` with this class inside the picker itself